src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV1.cs

```diff
@@ -56,34 +56,22 @@ public class PublishPipelineArtifactTaskV1 : PipelineArtifactTaskPluginBaseV1
         protected override string TargetPath => "path";
 
         private static readonly Regex jobIdentifierRgx = new Regex("[^a-zA-Z0-9 - .]", RegexOptions.Compiled | RegexOptions.CultureInvariant);
+        private static readonly string pipelineType = "pipeline";
```

Comment:
>static readonly string [](start = 16, length = 22)

in favor of changing these to consts (I'm not even sure why all the string constants in the agent repo are static readonly)

Reply:
Please check the full code logic. it's getting file from all levels